listenbrainz/mbid_mapping/docker/consul_config.py.ctmpl

```diff
@@ -8,14 +8,37 @@ MBID_MAPPING_DATABASE_URI = "dbname=musicbrainz_json_dump user=musicbrainz host=
 {{end}}
 {{end}}
 
+SQLALCHEMY_DATABASE_URI = ""
+MESSYBRAINZ_SQLALCHEMY_DATABASE_URI = ""
+POSTGRES_ADMIN_URI = ""
+MB_DATABASE_MASTER_URI = ""
+
 {{if service "pgbouncer-master"}}
 {{with index (service "pgbouncer-master") 0}}
 SQLALCHEMY_DATABASE_URI = "postgresql://listenbrainz:listenbrainz@{{.Address}}:{{.Port}}/listenbrainz"
 MESSYBRAINZ_SQLALCHEMY_DATABASE_URI = "postgresql://messybrainz:messybrainz@{{.Address}}:{{.Port}}/messybrainz"
-POSTGRES_ADMIN_URI="postgresql://postgres@{{.Address}}:{{.Port}}/postgres"
+POSTGRES_ADMIN_URI = "postgresql://postgres@{{.Address}}:{{.Port}}/postgres"
+MB_DATABASE_MASTER_URI = "postgresql://mbid_mapper@{{.Address}}:{{.Port}}/musicbrainz_db"
 {{end}}
 {{else}}
 SQLALCHEMY_DATABASE_URI = "SERVICEDOESNOTEXIST_pgbouncer-master"
+MESSYBRAINZ_SQLALCHEMY_DATABASE_URI = "SERVICEDOESNOTEXIST_pgbouncer-master"
+POSTGRES_ADMIN_URI = "SERVICEDOESNOTEXIST_pgbouncer-master"
+MB_DATABASE_MASTER_URI = "SERVICEDOESNOTEXIST_pgbouncer-master"
+{{end}}
+
+MB_DATABASE_STANDBY_URI = ""
+
+{{if service "pgbouncer-slave"}}
```

Comment:
For some reason, I am awestruck that this sort of config is possible. I love our infrastructure. :)